NEWS.md

```diff
@@ -16,6 +16,10 @@ for human readability.
   [SummationByPartsOperators.jl](https://github.com/ranocha/SummationByPartsOperators.jl)
   as approximation type in `DGMulti` solvers
 
+#### Removed
+
+- The `VertexMappedMesh` type was removed in favor of the `DGMultiMesh` type.
```

Comment:
```suggestion
- The `VertexMappedMesh` type is removed in favor of the `DGMultiMesh` type.
```
Should it be present tense, as in "these are the changes that are applied with the new release" (as opposed to the more passive "these changes have been applied during this release cycle")? I always flip flop on this though, so just make sure you stick to the prevailing style ;-)

Reply:
I have no idea TBH 😅. I'll go with present tense if it's consistent with previous announcements.